modules/nf-core/custom/matrixfilter/tests/main.nf.test.snap

```diff
@@ -0,0 +1,95 @@
+{
+    "test_custom_matrixfilter": {
+        "content": [
+            [
+                [
+                    {
+                        "id": "test"
+                    },
+                    "test.filtered.tsv:md5,ecc3ca4c6538f9248dce844108474a66"
+                ]
+            ],
+            [
+                [
+                    {
+                        "id": "test"
+                    },
+                    "test.tests.tsv:md5,9a0f14988df8c1b8975039eed91b85e2"
+                ]
+            ],
+            [
+                "versions.yml:md5,275e382be9ca5bdd30ff5d6fc4340616"
+            ]
+        ],
+        "meta": {
+            "nf-test": "0.9.2",
+            "nextflow": "24.10.2"
+        },
+        "timestamp": "2024-12-13T21:21:11.862199262"
+    },
```

Comment:
From here it seems to be as if the R script takes the prefix (here the id = test) and uses this for the tsv files.

Reply:
ok thats fine :)